src/components/list/index.tsx

```diff
@@ -1,7 +1,20 @@
+import React from "react";
+
 export function List() {
     return (
-        <div>
-            List
-        </div>
-    )
+      <div>
+        <ul className="nav nav-tabs">
+          <li className="nav-item">
```

Comment:
list component need not to be aware of contacts or groups, it should jus accept a list in props and render it dynamically